treeswift/Tree.py

```diff
@@ -6,6 +6,7 @@
 from math import ceil,log
 from os.path import expanduser,isfile
 from warnings import warn
+from tqdm import tqdm
```

Comment:
No non-built-in imports unless necessary (e.g. matplotlib, which is also optional)